### PR TITLE
MOTOR-1266 Update path to CSFLE scripts

### PR DESCRIPTION
**Pull request by blink1073**

### CODE REVIEW

**Review by ShaneHarvey — APPROVED** [review]

File: .evergreen/config.yml [hunk 1]
```diff
@@ -343,20 +343,20 @@ functions:
...
     - command: subprocess.exec
       params:
         working_dir: "src"
         binary: bash
         args:
-          - ${DRIVERS_TOOLS}/.evergreen/csfle/await_servers.sh
+          - ${DRIVERS_TOOLS}/.evergreen/csfle/await-servers.sh
     - command: shell.exec
       type: test
       params:
```
Comment (ShaneHarvey) [inline .evergreen/config.yml:362]: Also on line 387

**Review by ShaneHarvey — CHANGES_REQUESTED** [review]

File: .evergreen/config.yml (hunk 1 above)
Comment (blink1073) [reply .evergreen/config.yml:362]: Thanks, updated

**Review by ShaneHarvey — APPROVED** [review]

File: .evergreen/config.yml [hunk 2]
```diff
@@ -343,20 +343,20 @@ functions:
         binary: bash
         include_expansions_in_env: ["AWS_ACCESS_KEY_ID", "AWS_SECRET_ACCESS_KEY", "AWS_SESSION_TOKEN"]
         args:
-          - ${DRIVERS_TOOLS}/.evergreen/csfle/setup_secrets.sh
+          - ${DRIVERS_TOOLS}/.evergreen/csfle/setup-secrets.sh
```
Comment (ShaneHarvey) [inline .evergreen/config.yml:346]: Should this be secrets_handling/?

**Review by ShaneHarvey — CHANGES_REQUESTED** [review]

File: .evergreen/config.yml (hunk 2 above)
Comment (blink1073) [reply .evergreen/config.yml:346]: No, we only call secrets-handling when there is no convenience script already (such as for enterprise auth).

**Review by ShaneHarvey — APPROVED** [review]

**Comment by ShaneHarvey** [comment]
Lgtm for real this time :)

**Pull request merged**